### PR TITLE
RUN-4398: Enable chrome pages

### PR DESCRIPTION
**Pull request by wenjunche**
Tests

[W7](https://testing-dashboard.openfin.co/#/app/sessions/api/completed/5b96bbd56107b45e6d1ebc5b)
[W10](https://testing-dashboard.openfin.co/#/app/sessions/api/completed/5b96bce86107b45e6d1ebc5c)

### CODE REVIEW

**Comment by HarsimranSingh** [comment]
@wenjunche you can use this now that it's merged https://github.com/openfin/runtime/pull/1162

**Comment by datamadic** [comment]
🌳 This is blocked for now

File: src/common/main.ts [hunk 1]
```diff
@@ -2,6 +2,11 @@
 import { parse as parseUrl } from 'url';
 import { Identity } from '../shapes';
 
+export function isChromePageUrl(url: string): boolean {
+    const protocol = parseUrl(url).protocol || '';
```
Comment (whyn07m3) [inline src/common/main.ts]: :neckbeard:  `const { protocol = '' } = parseUrl(url);`

**Comment by wenjunche** [comment]
New tests
[W10](https://testing-dashboard.openfin.co/#/app/sessions/api/completed/5bc62eddcb360141a7dfd0b3)
[W7](https://testing-dashboard.openfin.co/#/app/sessions/api/completed/5bc62dc7cb360141a7dfd0b2)

File: src/browser/api/application.js [hunk 2]
```diff
@@ -1091,7 +1091,8 @@ function createAppObj(uuid, opts, configUrl = '') {
 
         opts.url = opts.url || 'about:blank';
 
-        if (!isHttpUrl(opts.url) && !isFileUrl(opts.url) && !opts.url.startsWith('about:') && !path.isAbsolute(opts.url)) {
+        if (!isChromePageUrl(opts.url) && !isHttpUrl(opts.url) && !isFileUrl(opts.url) &&
```
Comment (HarsimranSingh) [inline src/browser/api/application.js]: Can we do something to make this easier to read:
```js
const isAllowedChromePageUrl = process.buildFlags.enableChromium && isChromePageUrl(opts.url); // chrome pages are only allowed in enable_chromium builds
const isValidUrl = isHttpUrl(opts.url) || isFileUrl(opts.url) || opts.url.startsWith('about:') || path.isAbsolute(opts.url) || isAllowedChromePageUrl
if (!isValidUrl) {
  ...
}
```

File: src/browser/api/application.js [hunk 3]
```diff
@@ -31,7 +31,8 @@ import { validateNavigationRules } from '../navigation_validation';
 import * as log from '../log';
 import SubscriptionManager from '../subscription_manager';
 import route from '../../common/route';
-import { isFileUrl, isHttpUrl, getIdentityFromObject } from '../../common/main';
+import { isAboutPageUrl, isChromePageUrl, isFileUrl, isHttpUrl, isURLAllowed, getIdentityFromObject }
+from '../../common/main';
```
Comment (HarsimranSingh) [inline src/browser/api/application.js]: Why is this on a new line?

**Pull request merged**